pkg/models/monitoring/monitoring.go

```diff
@@ -182,7 +182,13 @@ func (mo monitoringOperator) GetKubeSphereStats() Metrics {
 		})
 	}
 
-	tmpls, err := mo.op.ListApps(&params.Conditions{}, "", false, 0, 0)
+	cond := &params.Conditions{
+		Match: map[string]string{
+			"status":  "active",
```

Comment:
Try not to hardcode these variables, it's bad practice

Reply:
they are fixed values

Reply:
done